### ISSUE TITLE
GitHub Actions Linter: `vars` context

### ISSUE THREAD

**Issue by ncalteen** [super-linter/super-linter#4106]
## Describe the bug

When linting GitHub Actions workflow files, Super Linter indicates the [`vars` context](https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context) is unavailable.

## Expected behavior

Using `${{ vars.VARIABLE_NAME }}` should not cause a linting error.

## Steps to Reproduce

1. Create a repository or organization Actions variable (e.g. `VAR_NAME`).
2. Create an example workflow file that uses the variable.
3. Lint the workflow file with Super Linter.

Example workflow file:

```yaml
name: Test

on:
  issues:
    types:
      - opened
      - edited

jobs:
  cause-error:
    name: Cause an Error
    runs-on: ubuntu-latest

    steps:
      - name: Post message
        uses: actions/github-script@v6
        with:
          script: |
            let myvar = '${{ vars.VAR_NAME }}'
```

## Logs

```plain
2023-04-18 16:49:06 [INFO]   ---------------------------
2023-04-18 16:49:06 [INFO]   File:[/github/workspace/scripts/migrations-via-actions-v3-from-bitbucket/.github/workflows/02-migration-bitbucket-server.yml]
2023-04-18 16:49:06 [ERROR]   Found errors in [actionlint] linter!
2023-04-18 16:49:06 [ERROR]   Error code: 1. Command output:
------
scripts/migrations-via-actions-v3-from-bitbucket/.github/workflows/02-migration-bitbucket-server.yml:13:11: could not read reusable workflow file for "./.github/workflows/03-shared-bitbucket-server-export.yml": open /github/workspace/.github/workflows/03-shared-bitbucket-server-export.yml: no such file or directory [workflow-call]
   |
13 |     uses: ./.github/workflows/03-shared-bitbucket-server-export.yml
   |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
scripts/migrations-via-actions-v3-from-bitbucket/.github/workflows/02-migration-bitbucket-server.yml:18:33: undefined variable "vars". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy" [expression]
```

## Additional context

Super Linter version: 4.10.0

**Comment by ncalteen** [comment]
Commenting to remove the stale label

**Comment by ncalteen** [comment]
Commenting to remove the stale label

**Comment by zkoppert** [comment]
👋🏻 Hi there, I'm unable to reproduce this issue with the latest version of the super-linter. I have given it a try with your sample workflow here: https://github.com/githubospo/test/pull/3 . It looks like this may be resolved by updating to the latest super-linter version which is `uses: super-linter/super-linter@v5` as shown here: https://github.com/githubospo/test/blob/main/.github/workflows/super-linter.yaml#L54

Feel free to reopen if this is not in fact resolved. Thanks!
